scene/resources/3d/primitive_meshes.cpp

```diff
@@ -966,6 +990,8 @@ void CylinderMesh::_create_mesh_array(Array &p_arr) const {
 void CylinderMesh::create_mesh_array(Array &p_arr, float top_radius, float bottom_radius, float height, int radial_segments, int rings, bool cap_top, bool cap_bottom, bool p_add_uv2, const float p_uv2_padding) {
 	int i, j, prevrow, thisrow, point;
 	float x, y, z, u, v, radius, radius_h;
+	const double EPSILON = 1e-10;
+
```

Comment:
```suggestion
	const double EPSILON = 1e-10;
```

Reply:
Also this isn't used anywhere is it?

Reply:
no, I forgot to remove it when testing for a fix.